src/main/scala/cromwell/logging/WorkflowLogger.scala

```diff
@@ -0,0 +1,80 @@
+package cromwell.logging
+
+import akka.event.LoggingAdapter
+import cromwell.engine.WorkflowDescriptor
+import org.slf4j.Logger
+
+/**
+ * This sets up two loggers for a given workflow, specified by the
+ * WorkflowDescriptor.
+ *
+ * 1) the Akka event logger (optional).  The log messages sent to the Akka
+ *    logger will propagate to the root logger (see logback.xml)
+ *
+ * 2) A logger that writes to a file that only has log messages
+ *    for this particular workflow.  This makes it easier to debug
+ *    a particular workflow if the messages aren't interleaved with
+ *    other workflows.  This logger does NOT propagate to the root
+ *    logger
+ */
+case class WorkflowLogger[T](descriptor: WorkflowDescriptor, akkaLogger: Option[LoggingAdapter] = None, otherLoggers: Seq[Logger] = Seq.empty[Logger]) {
```

Comment:
I don't know what the digestion utility is or how it helps.  I'm trying to fix this now because I need it for GitC, so I might have some questions for you @mcovarr